docs/src/components/github/SignUp.svelte

```diff
@@ -53,6 +53,11 @@
     background-color: #218838;
   }
 
+  a :global(svg) {
```

Comment:
quesiton: what is global doing? I never used it before?

Reply:
`:global` allows you to style things that aren't in the current template.  [See this link](https://joyofcode.xyz/global-styles-in-sveltekit).